### PR TITLE
Fixed Travis + Bumping versions

### PR DESCRIPTION
**Pull request by opticod**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -23,7 +23,7 @@ android {
         minSdkVersion 15
         targetSdkVersion 23
         versionCode 80
-        versionName "0.8.0"
+        versionName "0.9.0"
```
Comment (championswimmer) [inline app/build.gradle]: This is not be clubbed with any other change. 
Version bumps are NOT dependent on libs, but on the features we add to the app.

File: build.gradle [hunk 2]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.0.0-beta2'
+        classpath 'com.android.tools.build:gradle:1.5.0'
```
Comment (championswimmer) [inline build.gradle]: This will cause other conflicts, by going version down.

Comment (championswimmer) [inline build.gradle]: This will cause other conflicts, by going version down.

**Comment by championswimmer** [comment]
Done by this https://github.com/fossasia/open-event-android/pull/302 anyway

File: app/build.gradle (hunk 1 above)
Comment (championswimmer) [inline app/build.gradle]: Also we cannot go to 0.9.0 untill we solve all these 
https://github.com/fossasia/open-event-android/milestones/v0.9.0

Comment (opticod) [inline app/build.gradle]: Ok.

**Pull request closed without merging**